content/docs/for-buildpack-authors/how-to/distribute-buildpacks/craft-order.md

```diff
@@ -3,8 +3,16 @@ title="Craft a buildpack order"
 weight=99
 +++
 
+Buildpack order is
+
 <!--more-->
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+## Composite Buildpacks
+
+A **composite buildpack** is a buildpack that doesn't contain any `/bin/detect` or `/bin/build` executables; instead it references other buildpacks in its `buildpack.toml` via the [[order]] array. Composite buildpacks MUST be [resolvable](https://github.com/buildpacks/spec/blob/main/buildpack.md#order-resolution) into a collection of component buildpacks.
+
+## Why to use composite buildpacks
```

Comment:
Do we want to use Title Case?

```suggestion
## Why Use Composite Buildpacks
```

Reply:
updated

Reply:
I'd actually argue not to use title case, as I find that it's difficult to be consistent. But, as long as we're consistent within a particular page, that should be okay.